rotkehlchen/globaldb/handler.py

```diff
@@ -1750,7 +1772,8 @@ def resolve_asset(identifier: str, form_with_incomplete_data: bool) -> AssetWith
         UNION ALL
         SELECT A.identifier, A.type, null, null, A.name, null, null, null, null, null, null, null, null, null, B.notes, B.type FROM assets AS A JOIN custom_assets AS B on A.identifier=B.identifier WHERE A.identifier = ?
         """  # noqa: E501
-        with GlobalDBHandler().conn.read_ctx() as cursor:
+        connection = GlobalDBHandler().packaged_db_conn() if use_packaged_db is True else GlobalDBHandler().conn  # noqa: E501
```

Comment:
no this is not the only thing you need to do. 

If you succesfully resolve the asset from the packaged DB then since it's missing from the normal global DB you have to write it there too. You need code to do this.

And code should work for both `UnknownAsset` and `WrongAssetType`. Which mean you should delete the asset if it existed and then replace it with the global DB version.

If you don't do that do you realize what happens? Every single time you resolve the assets you will be going towards the global DB and would have chains of resolutions. Which is crazy

Reply:
If the type is modified this logic is never reached since is only triggered on `UnknownAsset`

Reply:
I don't understand what you mean. resolve_asset() can raise `WrongAssetType` which would get us to the packaged DB logic